docs/api-reference/geo-layers/tile-layer.md

```diff
@@ -138,10 +138,17 @@ if (signal.aborted) {
 
 The pixel dimension of the tiles, usually a power of 2.
 
-The tile size represents the target pixel width and height of each tile when rendered. Smaller tile sizes display the content at higher resolution, while the layer needs to load more tiles to fill the same viewport.
+For geospatial viewports, tile size represents the target pixel width and height of each tile when rendered. Smaller tile sizes display the content at higher resolution, while the layer needs to load more tiles to fill the same viewport.
```

Comment:
I gave this issue some more thought. It might be simpler if we just ask the user to always set `tileSize` to the true pixel size of the tiles.

- For geospatial views, `zoom:0` means the entire Web Mercator world is fit into 512x512. Therefore, in order to display raster tiles at a specified pixel size, we should adjust `zoom` in calculating `tile.z`.
- For non-geospatial views, `zoom:0` means 1 world units ~ 1 pixel, therefore no tileSize-dependent adjustment needs be made to z - i.e. your revert is correct.

Reply:
Ok @Pessimistress this makes sense with your other comments.  I'll fix these things and re-request review.  Thanks for thinking about this!

Reply:
@Pessimistress could you clarify what the first points means?  You're saying similar to what you're saying below, that `zoomOffset` should be used to adjust `zoom`?

Reply:
I was just reasoning why we have different logic for geo and non-geo. Basically keep the current geospatial logic but add `zoomOffset` to z.

Reply:
@Pessimistress Could you propose a change?  I am still not clear on what is being asked here, my apologies.